src/ucp/core/ucp_ep.h

```diff
@@ -84,8 +84,8 @@ typedef struct ucp_ep_config {
 
     /* Limits for active-message based protocols */
     struct {
-        size_t                 max_eager_short;  /* Maximal payload of eager short */
-        size_t                 max_short;        /* Maximal payload of am short */
+        ssize_t                max_eager_short;  /* Maximal payload of eager short */
+        ssize_t                max_short;        /* Maximal payload of am short */
```

Comment:
do we need to change the rest to ssize as well, so in future we have an option to bypass it ?